### DISCUSSION TITLE
Qortal as an Identity Provider

### DISCUSSION THREAD

**Discussion by QuickMythril** [gitcoinco/passport#1572] [Q&A]
I am a supporter of both Idena, recently added to the passport, as well as Qortal.  They are the only two blockchain projects I have found, aside from Neuralium which has been discontinued, which attempt to limit mass account farming, without requiring any sort of KYC.  Qortal requires new community members to get a "minting key" from an existing account, which is Level 5 or higher.  This is similar to how Idena validation first requires an "invite" from Verified/Human accounts.

I briefly mentioned this in the Gitcoin Discord, and seemed to get a positive response from @lucianHymer but I figured it would be appropriate to share this with everyone before starting to work on an implementation.  I'm happy to answer any questions from the Gitcoin community, or hear any concerns or thoughts on this idea.  I will start with my reasoning for the benefits of this collaboration.

- For the Qortal community, an overview of the Gitcoin Passport:
The Gitcoin Passport is a system that can aggregate account info from various identity providers and other services.  The idea is to use a variety of metrics to determine how likely it is for any given account to be a unique person or not.  For example, I would be more likely to give someone a minting key if they were able to show me an address they control, with a relatively higher passport score.  I would like to add Qortal as one of the providers, which would allow higher level Qortal accounts to be worth a certain number of points on the passport.  While we cannot completely ensure that any one person has only one minting account, I think it still indicates a degree of uniqueness.  Here is a list of the other platforms that are included, and their scores and requirements, for comparison.
https://support.gitcoin.co/gitcoin-knowledge-base/gitcoin-passport/common-questions/how-is-gitcoin-passports-score-calculated

- For the Gitcoin community, an overview of Qortal:
Qortal is a blockchain based platform, focusing on complete decentralization, which includes private or group chat, peer-to-peer websites and apps hosting, multi-asset wallets with native cross-chain trades (no wrapped coins), and more.  It's fully open for anyone to build anything they want, without owners, KYC, or censorship.  Only you can control your content, and choose what you want to see or not see.  The chain is secured by a custom consensus protocol that differs from both PoW and PoS, to prevent those with more resources (mining hardware, or coins to stake) from gaining more influence over the network.  To take part in consensus, you need to have a node running, with a minting key.  Anyone level 5 or higher can create a key to share with others, and this is meant to help the network grow organically, in a sense like a trust network.  To reach Level 5, an account has to be online and minting for 244,000 blocks.  With a block time of about 1 minute, they would be supporting the network for over a year.  Here is a somewhat more technical explanation of how the minting and leveling systems work in detail.
https://wiki.qortal.org/doku.php?id=how_qortal_s_consensus_works
https://wiki.qortal.org/doku.php?id=leveling_system

**Comment by teamak2020** [comment]
I get your concept, but considering Decentralization Gitcoin interfaces outside with Google, Facebook, Linkedin, Twitter and the list goes on, Are you trying to create a better, or higher form of authentication to the human? by using a outside "already verified" account? I gave it a upvote but would like to hear more.

**Reply by QuickMythril** [reply]
sure, i'm glad to elaborate.  let's say for example someone wants to start a new Proof of Stake chain, but there needs to be coins to start for people to stake.  sometimes they do an airdrop for a fair initial distribution.  ideally each person can only sign up once, so they need some way to verify people's identity, or at least uniqueness from any other account signing up.  usually there is some kind of requirement to link a verified twitter or other account, or giving some KYC.

gitcoin passport is an aggregate of different methods to verify an identity.  for example, the more centralized services like Facebook, Google, and Twitter have their own methods to fight bots.  i just tried to create a Facebook and they wanted my real name.  Quick Mythril did not work, so i tried Mythril Quick, and it accepted it, but i was "banned" on my next log in.  they wanted me to appeal and give a picture of my face, etc.

there are other more decentralized identity systems included on the passport as well.  Proof of Humanity requires you to upload a video of yourself, showing a wallet address, then other people confirm you are a human and not a bot or deepfake video, etc.  BrightID also requires you to join a video call, and have someone from their team verify you are an actual person.  these have no cost except time, but require KYC in the form of a video.

another one that's main goal is "Proof of Person" is Idena, which is a little more involved.  you have to attend a session at a specific time and complete some community-made captchas.  this one has no KYC or fees, but it does take more effort.  it's good to have different options for people with different preferences.  everyone has a different level of personal info they are comfortable to share.

so all of the things included on the passport are "outside already verified accounts" but by combining them, it creates a more flexible decentralized system.  none of them are perfect ways to limit an account to "one per person" but they all help to different degrees.  Qortal is not necessarily "higher or better authentication" but it's another free non-KYC option, for those who prefer that.  there's no way to be 100% sure that someone only has one minting account, but it's not easy to have as many as you want.  someone who can prove they own a level 5 Qortal account, PLUS a verified identity on Idena, and BrightID, and Proof of Humanity, is far more likely to be a unique individual.  separately they provide some sybil resistance, but together they are much stronger of a system.  i just want to add Qortal to that.

hopefully that explains what i am trying to accomplish and why, but if you still have questions, let me know.  :)

**Reply by teamak2020** [reply]
I do think it's important to verify the human aspect, as long as everything else is stripped away and burned in a burn barrel... we are headed in a great direction. No gooberment, or 3 letter, five eye or Grandma Allcorn should need to know what any other person looks like, or what GPS grid they are in, or triangulated cell position they happen to be passing thru.  We are sovereign only to God or the Creator, Amen

**Reply by gayashan091** [reply]
Nice concept

**Comment by raymonredington** [comment]
hmm

**Comment by Psk1one** [comment]
Idena its 🙈
